sandy/fy.py

```diff
@@ -504,10 +505,10 @@ def apply_bmatrix(self, zam, e, decay_data, keep_fy_index=False):
         data = data.loc[~mask]
         fy_data = fy_data.reindex(columns).fillna(0)
         cov_data = cov_data.reindex(columns).fillna(0)
-        cov_data = sandy.CategoryCov.from_var(cov_data)
+        cov_data = sandy.CategoryCov.from_stdev(cov_data)
```

Comment:
This is a covariance matrix. I would change the variable name to show it is not what cov_data was before.